content/blog/_posts/inclusion-good-for-business.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Making Inclusion Part of Your Business is Better for Business. Period."
+date: 2021-05-14
+slug: inclusive-good-for-business
+author: Wendy Renstchler
+---
+
+As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice. And more companies have recognized that one month a year doesn’t come close to being enough. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.
```

Comment:
~~As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice.  And more companies have recognized that one month a year doesn’t come close to being enough.~~ 
As we continue to advance into the age of technology, attention is being paid to how companies are responding to their learnings from the various social justice initiatives surrounding us.